### PR TITLE
[12.0][MIG] web_no_bubble: Migration to 12.0

### PR DESCRIPTION
**Pull request by Tardo**
Cc @Tecnativa

### CODE REVIEW

File: web_no_bubble/i18n/web_no_bubble.pot [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 11.0\n"
```
Comment (tbaden) [inline web_no_bubble/i18n/web_no_bubble.pot:6]: s/Odoo Server 11.0/Odoo Server 12.0

**Review by tbaden — APPROVED** [review]
thanks for the work 👍 
I did a code review: LGTM, just a minor details

File: web_no_bubble/i18n/web_no_bubble.pot (hunk 1 above)
Comment (pedrobaeza) [reply web_no_bubble/i18n/web_no_bubble.pot:6]: This is updated automatically from POT, so don't worry about this.

**Review by chienandalu — APPROVED** [review]
:+1: Tested on runbot Thanks!

**Pull request merged**